Dockerfile.test

```
@@ -1,35 +1,58 @@
 # docker build -t esn_sabre_test -f Dockerfile.test .
-#
 
-FROM debian:bullseye
+FROM debian:trixie-slim
 LABEL maintainer Linagora Folks <openpaas@linagora.com>
 
-ENV PHPVERSION=7.4
+ENV PHPVERSION=8.4
 
 # Copy PHP configuration files first (these rarely change)
 ADD ./docker/config/20-apcu.ini /etc/php/${PHPVERSION}/fpm/conf.d/20-apcu.ini
```

Hadolint (2.14.0)

[error] Use COPY instead of ADD for files and folders

(DL3020)